src/app/components/Image/index.tsx

```diff
@@ -0,0 +1,123 @@
+/** @jsx jsx */
+import React from 'react';
+import { jsx } from '@emotion/react';
+import { Helmet } from 'react-helmet';
+import styles from './index.styles';
+
+interface Props {
+  alt: string;
+  aspectRatio?: number;
+  className?: string;
+  fallbackMediaType: string;
+  fallbackSrcset: string;
+  height: number;
+  isAmp?: boolean;
+  lazyLoad?: boolean;
+  placeholder?: boolean;
+  preload?: boolean;
+  primaryMediaType: string;
+  primarySrcset: string;
+  sizes: string;
+  src: string;
+  width: number;
+}
+
+const DEFAULT_ASPECT_RATIO = 16 / 9;
+
+const Image = ({
+  alt,
+  aspectRatio,
+  className,
+  fallbackMediaType,
+  fallbackSrcset,
+  height,
+  isAmp = false,
+  lazyLoad = false,
+  placeholder = true,
+  preload = false,
+  primaryMediaType,
+  primarySrcset,
+  sizes = '100vw',
+  src,
+  width,
+}: React.PropsWithChildren<Props>) => {
```

Comment:
Minor thing but you could import this as `{PropsWithChildren} from react`, similar case below with `Fragment`.